### PR TITLE
Make review apps work. Maybe.

### LINKED ISSUE #1842: Enable review apps
It would be good (especially for designers) to test PRs without having to setup donate locally. Let's enable review apps for this repo.

This is definitely a nice to have and a lower priority than implementation issues.

### PR DESCRIPTION
**Pull request by alanmoo**
I think this closes #1842

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -4,10 +4,10 @@
   },
   "env": {
     "ADEPT_SCALE_LICENSE_KEY": {
-      "required": true
+      "required": false
```
Comment (gideonthomas) [inline app.json]: you should just remove these env vars because they aren't actually a part of staging.

**Comment by alanmoo** [comment]
Ok, removed them.

File: app.json [hunk 2]
```diff
@@ -3,12 +3,6 @@
   "scripts": {
   },
   "env": {
-    "ADEPT_SCALE_LICENSE_KEY": {
```
Comment (gideonthomas) [inline app.json]: also remove the adept scale add on from this json file as well

File: app.json [hunk 3]
```diff
@@ -3,12 +3,6 @@
...
-      "required": true
-    },
-    "ADEPT_SCALE_URL": {
-      "required": true
-    },
     "APPLICATION_URI": {
```
Comment (gideonthomas) [inline app.json]: this probably needs to be the review app application uri (i don't know if we rely on this for cors, etc.) but I don't know how to set that here. I think for now maybe we can just leave this as you have it.

**Pull request merged**